client/routes.js

```diff
@@ -40,6 +41,7 @@ export default (
         <Route path="/courses/:course/assignments/:assignment" component={Assignment} />
       </Route>
     </Route>
+    <Route path="*" component={ErrorPages} error="404" />
```

Comment:
Could you explain why you have to set error=404? I thought the reason for having a single errorpages component was that the errorpages would switch based on the actual error status

Reply:
so if you used this inside a api call then you would but this will be a 404 no matter what so i hard coded it.

Reply:
fair